msteams-platform/Teams-AI-library-tutorial.yml

```diff
@@ -38,56 +37,63 @@ items:
    | &nbsp; | [Visual Studio Code](https://code.visualstudio.com/download) | JavaScript, TypeScript, or SharePoint Framework (SPFx) build environments. Use the latest version. |
    | &nbsp; | [Azure OpenAI](https://oai.azure.com/portal)| OpenAI's GPT. Ensure that you first create your OpenAI API key. To host your app or access resources in Azure, you must create an Azure OpenAI service. |
 
+    [!INCLUDE [Set up prerequisites](includes/custom-engine-agent.md)]
+
 - title: Set up your data on Azure OpenAI
   durationInMinutes: 1
   content: | 
-     1. Go to [**Azure portal**](https://ms.portal.azure.com/).
 
-     1. Select the Azure OpenAI service that you created.
+   /**After you set up your project workspace with Teams Toolkit, you need to create an Azure OpenAI key using Azure portal./
+
+    ## Create an Azure AI Search endpoint and OpenAI key
+
+    1. Go to [**Azure portal**](https://ms.portal.azure.com/).
+
+    1. Select the Azure OpenAI service that you created.
 
-     1. Select **Go to Azure OpenAI Studio**.
+    1. Select **Go to Azure OpenAI Studio**.
         
-        :::image type="content" source="../msteams-platform/assets/images/bots/go-to-azure-openai-studio.png" alt-text="Screenshot shows the go to Azure OpenAI studio in Azure portal.":::
+      :::image type="content" source="../msteams-platform/assets/images/bots/go-to-azure-openai-studio.png" alt-text="Screenshot shows the go to Azure OpenAI studio in Azure portal.":::
```

Comment:
The UI and branding on screenshots are outdated. Azure AI Studio is now rebranded as Open AI Foundry. Can you use the screenshots for the current UI?

Reply:
This screenshot update will affect many documents across the repository as it is a significant change. It will be taken as a fast follow up update.